circuits/cpp/barretenberg/cpp/src/barretenberg/crypto/aes128/c_bind.cpp

```diff
@@ -1,8 +1,5 @@
 #include "aes128.hpp"
-
-#define WASM_EXPORT __attribute__((visibility("default")))
```

Comment:
I cleaned up the places WASM_EXPORT was used while untangling errors. This now takes advantage that 'extern C' is implied by WASM_EXPORT.